RNA-seq/05-nb_cell_line_DESeq2.Rmd

```diff
@@ -207,6 +207,57 @@ summary(results(deseq_object), alpha = 0.05)
 readr::write_tsv(deseq_df, path = deseq_df_file)
 ```
 
+#### Visualize differential expression results with a heatmap
+
+Let's get an overall picture of our results using a heatmap. 
+Before we can do this, we will need to transform our data so that when we make
+the heatmap the genes are on a more similar scale and the heatmap is 
+interpretable. 
+We will use variance stablizing transformation (vst) from DESeq2 to do this.
+Refer to the help page or the DESeq2 docs for more information. 
+
+```{r}
+vst_data <- vst(ddset)
+```
+
+Need to extract the normalized gene matrix from the `vst_data` object. 
+
+```{r}
+vst_data <- assay(vst_data)
+```
+
+~37,000 genes will be too crowded for one heatmap, let's calculate the standard
+deviation for all the genes using `apply` and then use `dplyr::filter` to keep 
+only the highest variance genes for this heatmap. 
```

Comment:
Do you think using the term variance here and calculating the standard deviation will muddy the point a bit?